tests/test-addons.py

```diff
@@ -110,6 +116,12 @@ def test_basic(self):
         Sets up and tests dashboard, dns, storage, registry, ingress, metrics server.
 
         """
+        # Set labels
+        node_name = kubectl(f"get node -o jsonpath='{{.items[0].metadata.name}}'")
```

Comment:
Oh yes. It can easily be a false positive. I was just getting the right name so didn't bother.
A cleaner approach can be reading the name from `/etc/hostname`. I will do that.